### PR TITLE
add image boot time test

### PR DESCRIPTION
**Pull request by gaohannk**

### CODE REVIEW

File: imagetest/test_suites/image_boot/image_boot_test.go [hunk 1]
```diff
@@ -71,3 +73,15 @@ func TestGuestSecureBoot(t *testing.T) {
...
+func TestBootTime(t *testing.T) {
+	metadata, err := utils.GetMetadataAttribute("start-time")
+	if err != nil {
+		t.Fatalf("couldn't get start time from metadata")
+	}
+	startTime, err := strconv.Atoi(metadata)
+	if err != nil {
+		t.Fatalf("failed to convet start time %s", metadata)
+	}
+	t.Logf("image boot time is %d", time.Now().Second()-startTime)
```
Comment (hopkiw) [inline imagetest/test_suites/image_boot/image_boot_test.go:86]: we don't care how long it took?

Comment (gaohannk) [reply imagetest/test_suites/image_boot/image_boot_test.go:86]: That's an open question. I didn't find a SLO for instance boot up.

Comment (gaohannk) [reply imagetest/test_suites/image_boot/image_boot_test.go:86]: So i make this is just for log purpose

Comment (gaohannk) [reply imagetest/test_suites/image_boot/image_boot_test.go]: Just a simple test point
```
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url: === RUN   TestBootTime
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url:     image_boot_test.go:86: image boot time is 26
Jul 27 19:58:49 vm2-image-boot-vd5tv GCEMetadataScripts[480]: 2021/07/27 19:58:49 GCEMetadataScripts: startup-script-url: --- PASS: TestBootTime (0.00s)
```
I make the threshold is 120 seconds.

**Pull request merged**